README.md

```diff
@@ -816,6 +817,28 @@ customButtons: [{
 
 - **customButtons.click** - Function that is called when the button is clicked
 
+## Adding Registration Link
+
+You can add registration link underneath the login form on the primary auth page by setting the following config options. If you'd like to change the label text and link text use the `i18n` config option.
+
+```javascript
+// An example that adds a custom button underneath the login form on the primary auth page
+registration: {
+  click: function( ){
+    window.location.href = 'http://www.google.com'
+  },
+  text: 'Register',
+  label: 'Dont have an account ?'
+}
+```
+- **registration.showLink** - Boolean that toggles show/hide of the registration section
+
+- **registration.linkUrl** - Link url
```

Comment:
Same with this - we don't need this. We're already showing them an example of a redirect on a click, which handles both cases.